### PR TITLE
Fast approach on a 4.x upgrade

### PR DESCRIPTION
**Pull request by markstory**
This gets all the tests passing against CakePHP 4.1. I've also run `phpcbf` with the 4.x standard which has added `strict_types` and done some other formatting changes. I've not invested the time to add additional typehints across the plugin. I can spend some additional time on that if you'd like.

### CODE REVIEW

**Comment by ishanvyas22** [comment]
First of all thank you for the PR. Sure, I would really love the typehints across the plugin.

**Comment by markstory** [comment]
> First of all thank you for the PR. Sure, I would really love the typehints across the plugin.

Welcome, Thanks for making this plugin :smile: I wanted to try inertia out and this plugin seems like a good approach to get it working. 

Would you be ok with additional typehints being added in a separate pull request, or would you want them in this one?

**Comment by ishanvyas22** [comment]
Let's do this in separate PR. I have separated `cake3` branch for CakePHP v3. Once the travis build is passing we are ready to merge this in.

File: src/Middleware/InertiaMiddleware.php [hunk 1]
```diff
@@ -2,32 +2,34 @@
```
Comment (ishanvyas22) [inline src/Middleware/InertiaMiddleware.php:3]: Travis build is failing due to missing `declare(strict_types=1)` here

Comment (ishanvyas22) [reply src/Middleware/InertiaMiddleware.php:3]: I've added it

**Review by ishanvyas22 — DISMISSED** [review]

**Review by ishanvyas22 — APPROVED** [review]

**Comment by ishanvyas22** [comment]
Mark, I've  also created https://github.com/ishanvyas22/cakephp-pingcrm to demonstrate this plugin into action. Might interest you!

**Pull request merged**